client/src/routes/Publish/Files/Item.module.scss

```diff
@@ -26,11 +26,11 @@
     position: absolute;
     top: $spacer / 8;
     right: 0;
-    transition: .2s ease-out;
+    transition: 0.2s ease-out;
 
     &:hover,
     &:focus {
-        opacity: .7;
+        opacity: 0.7;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5548133)